fun/envs/lms/unittests.py

```diff
@@ -7,6 +7,12 @@
 import logging
 logging.getLogger("backoffice.views").setLevel(logging.ERROR)
 logging.getLogger("edxmako.shortcuts").setLevel(logging.ERROR)
+logging.getLogger("openedx.core.djangoapps.content.course_overviews.models").setLevel(logging.ERROR)
+logging.getLogger("py.warnings").setLevel(logging.ERROR)
+logging.getLogger("backoffice.utils_proctorU_api").setLevel(logging.ERROR)
+logging.getLogger("elasticsearch").setLevel(logging.ERROR)
+logging.getLogger("urllib3.connectionpool").setLevel(logging.ERROR)
+logging.getLogger("payment_api.views").setLevel(logging.ERROR)
```

Comment:
This should be configured in Test environment settings.

Reply:
What do you mean ? This is the Django settings file to use to run unit tests

Reply:
You logs configuration should stand in the settings used with the test runner: https://docs.djangoproject.com/en/1.11/topics/logging/#configuring-logging